app/client/components/pages/transferAccountListPage.jsx

```diff
@@ -41,37 +41,25 @@ class TransferAccountListPage extends React.Component {
   }
 
   buildFilterForAPI() {
-      if (location.pathname.includes('vendors')) {
-          var query = {account_type: 'vendor'};
+    let query = {};
 
-      } else if (location.pathname.includes(window.BENEFICIARY_TERM_PLURAL.toLowerCase())) {
-          query = {account_type: 'beneficiary'};
-
-      } else {
-          query = {};
-      }
-
-      if (this.props.transferAccounts.loadStatus.lastQueried) {
-        query.updated_after = this.props.transferAccounts.loadStatus.lastQueried;
-      }
+    if (this.props.transferAccounts.loadStatus.lastQueried) {
+      query.updated_after = this.props.transferAccounts.loadStatus.lastQueried;
+    }
 
 
-      const path = null;
-      this.props.loadTransferAccountList(query, path);
+    const path = null;
+    this.props.loadTransferAccountList(query, path);
   }
   
   render() {
-    const is_vendor = location.pathname.includes('vendors');
-    const is_beneficiary = location.pathname.includes(window.BENEFICIARY_TERM_PLURAL.toLowerCase());
-
-    if (is_vendor || is_beneficiary) {
-        // just vendors or beneficiaries
-        var transferAccountList = this.props.mergedTransferAccountUserList.filter(transferAccount => {return transferAccount.is_vendor === is_vendor})
-    } else {
-        // all transfer accounts
-        transferAccountList = this.props.mergedTransferAccountUserList
+    let transferAccountList = this.props.mergedTransferAccountUserList;
+
+    if (this.props.login.adminTier === 'view') {
```

Comment:
not _super happy_ with string comparison, but we do it all over the codebase for adminTiers so seems OK here.

Reply:
I think we could start leaning harder on consts files. I know there's `constants.py`, but that's mostly lists/maps-- I'm taking something literally as simple as `const.VIEW = 'view'`.

Reply:
bigg second on that @michielderoos ^^
i don't mind refactoring that as my next task

Reply:
yeah a frontend contestants file would be great